1.6.0-rc2: compile error --with-java

Out of tree build.  Compiles OK if download the raw source asset and autogen.sh it.

```shell
Making all in bindings/java/src/main/native
make[2]: Entering directory '/usr/local/src/ucx/build/bindings/java/src/main/native'
mvn -B -f ../../../../../../ucx-1.6.0-rc2/bindings/java/pom.xml -Dmaven.repo.local=/usr/local/src/ucx/build/.deps/ compile native:javah
POM file ../../../../../../ucx-1.6.0-rc2/bindings/java/pom.xml specified with the -f/--file command line argument does not exist
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-readable POM /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0-rc2/bindings/java/pom.xml: /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0-rc2/bindings/java/pom.xml (No such file or directory) @
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project  (/usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0-rc2/bindings/java/pom.xml) has 1 error
[ERROR]     Non-readable POM /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0-rc2/bindings/java/pom.xml: /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0-rc2/bindings/java/pom.xml (No such file or directory)
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
make[2]: *** [Makefile:858: org_ucx_jucx_ucp_UcpContext.h] Error 1
make[2]: Leaving directory '/usr/local/src/ucx/build/bindings/java/src/main/native'
make[1]: *** [Makefile:722: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/src/ucx/build'
make: *** [Makefile:590: all] Error 2
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -47,9 +47,9 @@ endif
 
 if HAVE_JAVA
 SUBDIRS += bindings/java/src/main/native
-EXTRA_DIST += bindings/java
 endif
 
+EXTRA_DIST += bindings/java
 EXTRA_DIST += contrib/configure-devel
 EXTRA_DIST += contrib/configure-release
 EXTRA_DIST += contrib/configure-prof

